src/Components/Publishing/Icon/IconPlus.tsx

```diff
@@ -0,0 +1,34 @@
+import React, { Component } from "react"
+import styled from "styled-components"
+import { pMedia } from "../../Helpers"
+
+export class IconPlus extends Component<any, null> {
+  render() {
```

Comment:
Would be good to include a color prop here -- on superArticles it is rendered in white

Reply:
Noted, I'll add this in another time since it's not required for this project.